packages/client/test/db.e2e.spec.ts

```diff
@@ -340,6 +344,28 @@ describe('DatabaseResolver', () => {
       expect(count).eq(1)
     })
 
+    it('should allow register a domain with different owner', async () => {
+      const newOwner = privateKeyToAddress(generatePrivateKey())
+      const name = normalize('newdomain.eth')
+      const node = namehash(name)
+      const response = await offchainWriting({
```

Comment:
Is it registering a 2LD?

Reply:
Nope, this is a subdomain being created in the database. see [this line](https://github.com/blockful-io/external-resolver/pull/157/files/822557e91184a0fd336425608940b6b3e36ea362#diff-30d3acbdcd6476dcc99f9f2435b2e32631987d3395e12406fc68096f5fc903bcR260)